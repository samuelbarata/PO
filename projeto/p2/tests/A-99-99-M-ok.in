7
1
1
3
1
1
6
1
1
8
0
4
4
7
2
1
3
0
5
3
0
4
1
7
2
1
8
s
0
5
2
1
0
4
1
7
1
1
5
1
1
4
0
4
3
7
2
1
6
s
0
5
2
1
0
4
20
7
2
1
4
s
0
5
2
1
0
5
1
leya
leya@gmail.com
3
0
0