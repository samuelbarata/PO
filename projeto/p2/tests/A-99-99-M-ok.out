Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 7
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 1
Introduza o número de utente: 1
Introduza o número da obra: 3
A obra 3 deve ser devolvida no dia 15.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 1
Introduza o número de utente: 1
Introduza o número da obra: 6
A obra 6 deve ser devolvida no dia 8.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 1
Introduza o número de utente: 1
Introduza o número da obra: 8
A obra 8 deve ser devolvida no dia 3.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 4
Introduza número de dias a avançar: 4
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 7
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
Introduza o número da obra: 3
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 5
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 3
5 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
1 - Darth Vader - darth@dark.force.com - NORMAL - SUSPENSO - EUR 0
9 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
4 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
3 - Nome Excecivamente Grande que ninguem vai ler mas me apetece incerir aqui apenas pra testar - email@domain.com - NORMAL - ACTIVO
0 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
2 - Samuel Barata - email@domain.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 4
Introduza número de dias a avançar: 1
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 7
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
Introduza o número da obra: 8
O utente 1 deve pagar uma multa de EUR 10.
O utente deseja pagar multa (s/n)? s
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 5
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
1 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 4
Introduza número de dias a avançar: 1
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 7
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 1
Introduza o número de utente: 1
Introduza o número da obra: 5
Requisitar obra: Operação inválida: O utente 1 não pode requisitar a obra 5. Violação da regra 5.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 1
Introduza o número de utente: 1
Introduza o número da obra: 4
A obra 4 deve ser devolvida no dia 21.
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 4
Introduza número de dias a avançar: 3
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 7
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
Introduza o número da obra: 6
O utente 1 deve pagar uma multa de EUR 5.
O utente deseja pagar multa (s/n)? s
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 5
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
1 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 4
Introduza número de dias a avançar: 20
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 7
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
Introduza o número da obra: 4
O utente 1 deve pagar uma multa de EUR 40.
O utente deseja pagar multa (s/n)? s
Menu de Gestão de Requisições
1 - Requisitar obra
2 - Devolver obra
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 5
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 2
Introduza o número de utente: 1
1 - Darth Vader - darth@dark.force.com - FALTOSO - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 5
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 1
Introduza o nome do utente: leya
Introduza o endereço de correio do utente: leya@gmail.com
Novo utente criado com o número 10.
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 3
5 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
1 - Darth Vader - darth@dark.force.com - FALTOSO - ACTIVO
9 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
4 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
3 - Nome Excecivamente Grande que ninguem vai ler mas me apetece incerir aqui apenas pra testar - email@domain.com - NORMAL - ACTIVO
0 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
2 - Samuel Barata - email@domain.com - NORMAL - ACTIVO
10 - leya - leya@gmail.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 0